### PR TITLE
About_me and work experience commit

### PR DESCRIPTION
**Pull request by EngruugbeJimmy**
# In this milestone:
- aboutme and work experience added with best HTML and CSS practices
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,97 @@
...
+## Set-up GitHub Actions
 
-## Built With 🔨
+Please do the following **steps in this order**:
 
-- HTML
-- CSS
-- Git & Github
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
```
Comment (jaflih) [inline README.md]: Please update your readme file to follow [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md)

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @EngruugbeJimmy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed 💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EngruugbeJimmy** [comment]
# In this milestone;
-README.md file updated as requested

**Comment by EngruugbeJimmy** [comment]
# In this milestone
- -README.md file updated as requested

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @EngruugbeJimmy , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**